### PR TITLE
Deploy scraper for testnet3

### LINKED ISSUE #1348: Deploy the scraper to testnet3 environment
Helm charts and ts deploy code probably needs updating, see #1314 for examples of what was needed to get the agents running.

Consider #1344 if it makes things easier.

### PR DESCRIPTION
**Pull request by mattiecnvr**
### Description
* removed scraper-specific helm charts and merged into agent-common
* updated configs for testnet3
* fixed 0 addr bug in scraper

### Drive-by changes

* Simplified agent settings now that scraper is not a special case
* Updated seaorm
* Cleaned up scraper database tables
* Removed deprecated domains (since v2 does not need to have them for legacy support)
* Removed some code that somehow made it back into the v2 branch

### Related issues

- Fixes #1348

### Backward compatibility

_Are these changes backward compatible?_

No

_Are there any infrastructure implications, e.g. changes that would prohibit deploying older commits using this infra tooling?_

Yes

### Testing

_What kind of testing have these changes undergone?_

Manual

### CODE REVIEW

File: rust/Cargo.lock
Comment (mattiecnvr) [inline rust/Cargo.lock:100]: We will merge this into main once asaj/v2-deploy is merged. Comment to prevent auto-merge.

**Comment by mattiecnvr** [comment]
We will merge this into main once asaj/v2-deploy is merged.

**Review by tkporter — APPROVED** [review]
lgtm!

**Pull request merged**
